experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/table.html

```diff
@@ -109,11 +110,16 @@ <h1 class="modal-title fs-5" id="createModalLabel">
           </td>
           <td>
             {% if experiment.start_date %}
-              {{ experiment.start_date|date:"Y-m-d" }} -
-              <br>
-              {{ experiment.computed_end_date|date:"Y-m-d" }}
-              <br>
-              ({{ experiment.computed_duration_days }} days)
+              {{ experiment.start_date|date:"Y-m-d" }}
+            {% else %}
+              N/A
+            {% endif %}
+          </td>
+          <td>
+            {% if experiment.end_date %}
+              {{ experiment.end_date|date:"Y-m-d" }} ({{ experiment.computed_duration_days }} days)
+            {% elif experiment.proposed_end_date %}
+              <i>{{ experiment.proposed_end_date|date:"Y-m-d" }} ({{ experiment.computed_duration_days }} days)</i>
```

Comment:
Oh no, we should just display `computed_end_date` here.  If we start splitting up different logic then we'll wind up with different dates displayed in different places/pages and it gets confusing.

Reply:
Also I get your point about using italics here but we don't use that anywhere else to mean that, so I'd take that out for consistency.

Reply:
Yep I think I did this because I wasn't able to do sorting in the same way using computed properties, so I made the display consistent with the sorting. But if we can just sort in python where the `computed_end_date` is available then that's definitely better!

Also yes totally fine re: italics. Even less important if we land https://github.com/mozilla/experimenter/pull/11988